README.md

```diff
@@ -3,79 +3,177 @@
+Between different networks, Zenoh can search for nodes through a broker (called a router in Zenoh) like MQTT.
+Also, Zenoh provides functions for database operations and computational processing based on the Key-Value Store.
+Moreover, it has plugins/bridges for interoperability with MQTT, DDS, REST, etc. as communication middleware, and influxdb, RocksDB, etc. as database stacks.
 
-## Installation
+For more details about Zenoh, please refer to the official resources.
+- [Official Page](https://zenoh.io/)
+- [GitHub](https://github.com/eclipse-zenoh/zenoh)
+- [Discord](https://discord.gg/vSDSpqnbkm)
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `zenohex` to your list of dependencies in `mix.exs`:
+Zenoh's core modules are implemented in Rust, but API libraries in various programming languages such as Python ([zenoh-python](https://github.com/eclipse-zenoh/zenoh-python)), C ([zenoh-c](https://github.com/eclipse-zenoh/zenoh-c)), C++ ([zenoh-cpp](https://github.com/eclipse-zenoh/zenoh-cpp)) are officially provided.
 
-```elixir
-def deps do
-  [
-    {:zenohex, "~> 0.2.0-rc.2"}
-  ]
-end
-```
+So what we need is [Elixir](https://elixir-lang.org/)!
+With this library, you can call Zenoh from your Elixir application to perform its basic processing.
+This allows the creation and communication of a large number of fault-tolerant nodes with little memory load (we hope :D
+
+## Usage
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at <https://hexdocs.pm/zenohex>.
+**Currently, Zenohex uses version 0.10.1-rc of Zenoh.  
+We recommend you use the same version to communicate with other Zenoh clients or routers.**
 
-### Install rust (Optional)
+### Installation
 
-Since version 0.1.3, Zenohex uses rustler precompiled and does not require Rust to be installed.
+`zenohex` is [available in Hex](https://hex.pm/packages/zenohex).
 
-If you want to build rust NIFs code, please add following to your config file.
+You can install this package into your project by adding `zenohex` to your list of dependencies in `mix.exs`:
 
 ```elixir
-config :rustler_precompiled, :force_build, zenohex: true
+  defp deps do
+    [
+      ...
+      {:zenohex, "~> 0.2.0-rc.2"},
+      ...
+    ]
+  end
 ```
 
-https://www.rust-lang.org/tools/install
+Documentation is also [available in HexDocs](https://hexdocs.pm/zenohex).
+
+Zenohex can be also adapted to your [Nerves](https://nerves-project.org/) application just by adding `zenohex` in `mix.exs`.
+Please refer to [pojiro/nerves_zenohex](https://github.com/pojiro/nerves_zenohex) as the example.
 
-## Getting Started
+This repository uses [Rustler](https://github.com/rusterlium/rustler) to call Rust (Zenoh) modules from Elixir, and pre-compiled NIF modules are automatically downloaded at `mix compile` time (since v0.1.3). 
```

Comment:
```suggestion
This repository uses [Rustler](https://github.com/rusterlium/rustler) to call Rust (Zenoh) modules from Elixir, and pre-compiled NIF modules are automatically downloaded at `mix compile` time (since v0.1.3).
```

prettier で見つけた不要なスペースを削除